cfme/scripting/quickstart.py

```diff
@@ -280,21 +286,22 @@ def install_requirements(venv_path, quiet=False):
         sys.exit("ERROR: {} is required to be a file".format(remember_file))
     else:
         last_hash = None
-    if last_hash == current_hash:
+    if last_hash == current_hash and not PY3:
         print("INFO: skipping requirement installation as frozen ones didn't change")
         print("      to enforce please invoke pip manually")
         return
 
     elif last_hash is not None:
         current_packages = pip_json_list(venv_path)
         print("INFO:", REQUIREMENT_FILE, 'changed, updating virtualenv')
-
+    elif PY3:
```

Comment:
**REQUIRED:** Does this mean that at every invocation of ```miq shell``` we will do an install?

Reply:
for now it does - fixing freezing will elevate it